### ISSUE TITLE
Webm movie format not usable directly in powerpoint

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/three.js#10]
Default format for movies is webm which plays nicely in browsers but is not supported in Microsoft Powerpoint. Need to export other formats or convert on the GUI/Java side.

**Comment by aymanhab** [comment]
The .webm format has the advantage of being client/browser side so has less wiring to a server, but there are solutions that send snapshots to the server (GUI) which converts them to more popular formats. Will investigate

**Comment by chrisdembia** [comment]
Based on some reading I did a week ago, `mp4` (H.264 video encoding) seems good and cross-platform and works in PPT (http://arstechnica.com/civis/viewtopic.php?f=6&t=1119097).

**Comment by aymanhab** [comment]
Just for the record https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats
claims mp4 is not royalty free and that's why it's "not supported by Internet Explorer and Safari, Mozilla has decided to support the format anyway, and Google never fulfilled their promise to remove support for it in Chrome.".

**Comment by chrisdembia** [comment]
Oh thanks for that info; I did not realize. These media format issues sound like such a mess. Despite that sentence, the table says that MP4 is supported in all the listed browsers. Still, it seems like it would be good to support royalty-free formats. 

Perhaps we should support multiple formats?

**Comment by aymanhab** [comment]
Indeed, I'm still looking into the tools available to create mp4 but also avi (which we used to create in the past and no body complained about).

**Comment by chrisdembia** [comment]
I think it would be fine for us to support AVI, but I feel that it may only be possible on Windows. For example, I think that VTK on OSX does not contain the AVIFIleWriter or whatnot.

**Comment by sherm1** [comment]
My understanding of AVI is that it is a wrapper for many different underlying encodings. The chances that an AVI will play, even on Windows, are generally slim. Many of the AVIs I generated a few years ago on an earlier Windows machine won't play on my current Windows machine!

I think mp4 has a lot more promise as a universal format if there is a good way to generate it. Could always wait until the black helicopters swoop down to remove it. Likely to be a long wait!

**Comment by aymanhab** [comment]
Thanks for chiming in @sherm1 Indeed looking up comparisons between MP4 and AVI reveals that MP4 is cross-platform while AVI was developed by Microsoft and mainly on Windows, not good enough for the brave new cross-platform visualization. Cheers.

**Comment by chrisdembia** [comment]
Hello @sherm1!
